### PR TITLE
Add delete history to exploding messages

### PR DESCRIPTION
**Pull request by buoyad**
r? @keybase/react-hackers

### CODE REVIEW

File: shared/chat/conversation/messages/message-popup/exploding/container.js [hunk 1]
```diff
@@ -19,12 +20,12 @@ type OwnProps = {
 const mapStateToProps = (state: TypedState, ownProps: OwnProps) => {
   const yourMessage = ownProps.message.author === state.config.username
   const meta = Constants.getMeta(state, ownProps.message.conversationIDKey)
-  const canExplodeNow =
-    meta.teamType === 'adhoc' ||
-    yourMessage ||
-    TeamConstants.getCanPerform(state, meta.teamname).manageMembers
+  const canDeleteHistory =
+    meta.teamType === 'adhoc' || TeamConstants.getCanPerform(state, meta.teamname).deleteChatHistory
```
Comment (chrisnojima) [inline shared/chat/conversation/messages/message-popup/exploding/container.js]: Can perform can't be used in adhoc? We could make it right

Comment (buoyad) [reply shared/chat/conversation/messages/message-popup/exploding/container.js]: Everyone can do everything in adhoc conversations right?

**Comment by chrisnojima** [comment]
Yes but can user perform could just return those flags instead of everyone
always checking

**Comment by buoyad** [comment]
That might be useful. Existing code about this is pretty team specific and in the teams store etc so it might be a big change. Merging this for now

**Pull request merged**